### PR TITLE
Update changes from review

### PR DESCRIPTION
**Pull request by lmzuccarelli**
Updated  as suggested

/assign @darshan-nagaraj

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-# NodeObservability Operator [WIP]
+# NodeObservability Operator
 
-The `NodeObservability` Operator allows you to deploy and manage [NodeObservability Agent](https://github.com/openshift/node-observability-agent), these agents will be deployed as DaemonSets (on all or slected Nodes). \
-The agent is used to trigger crio and kubelet profile data to the nodes hostPath for later retrieval (this is the MVP). 
+The NodeObservability Operator allows you to deploy and manage [NodeObservability Agent](https://github.com/openshift/node-observability-agent) on worker nodes. The agent is deployed through DaemonSets on all or selected nodes. It also triggers the crio and kubelet profile data to the nodes hostPath for later retrieval.
 
 **Note**: This Operator is in the early stages of implementation, and will be changing.
```
Comment (darshan-nagaraj) [inline README.md:6]: ## Deploying the `NodeObservability` Operator

You can deploy the NodeObservability Operator for a BareMetal installation by using the following procedure:

### Installing the NodeObservability Operator

You can install the `NodeObservability` Operator by building and pushing the Operator image into a registry.

1. To build and push the Operator image into a registry, run the following commands:
   ```sh
   # set the envar CONTAINER_ENGINE to the preffered container manager tool (default is podman)
   $ export IMG=<registry>/<username>/node-observability-operator:latest
   $ make container-build
   $ make container-push
   ```
2. To deploy the `NodeObservability` Operator, run the following command:
   `$ make deploy`

### Creating the local NodeObservability

1. To create make targets, run the following command:
  ```sh
  $ make install
  $ make run
  # In another terminal execute the sample CR
  $ oc apply -f /config/samples/nodeobservability_v1alpha1_nodeobservability-all.yaml
  # Alternatevely you can create a new CR and change the fields accordingly
  ```

Comment (lmzuccarelli) [reply README.md:6]: Applied as suggested

**Comment by lmzuccarelli** [comment]
/assign @josefkarasek 
/assign @sherine-k 
/assign @ayesha54 

I have made changes for the README (darshan has already reviewed). This should have been a separate commit but I mistakenly included it. 

The changes really were to enable the goimports` local-prefixes: github.com/openshift/node-observability-operator` and ensure all imports would then pass the golangci-lint run.

File: cmd/node-observability-operator/main.go [hunk 2]
```diff
@@ -22,20 +22,21 @@ import (
...
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
 	//+kubebuilder:scaffold:imports
+
+	nodeobservabilityv1alpha1 "github.com/openshift/node-observability-operator/api/v1alpha1"
+	controller "github.com/openshift/node-observability-operator/pkg/operator/controller/nodeobservability"
```
Comment (josefkarasek) [inline cmd/node-observability-operator/main.go:38]: inconsistent naming

**Review by josefkarasek — APPROVED** [review]
Looks good, thank you

**Pull request merged**